src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/design/mutableexception/Example1.java

```diff
@@ -0,0 +1,47 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="MutableException"/>
+  </module>
+</module>
+*/
+
+
+
+package com.puppycrawl.tools.checkstyle.checks.design.mutableexception;
+
+// xdoc section -- start
+class Example1 extends Exception {
+  private int code; // OK, class name doesn't match with default pattern
+
+  public Example1() {
+    code = 1;
+  }
+}
+
+class FirstMyException extends Exception {
```

Comment:
'My' here is redundant and looks weird. Please change `FirstMyException` --> `FirstException`
Same for all

Reply:
Done